example/README.md

```diff
@@ -0,0 +1,51 @@
+Example Webpack setup
+======================
+
+Here you'll find an example Webpack configuration. The `webpack.config.js` here will output two bundles here: `application.js` and `vendor.js`, and will process an image.
+
+## Starting
+
+Clone the plugin and run `npm install` in the root folder and in the `example/` folder. Then start `npm run` to start Webpack. In the folder `build/` you should see four files:
+
+1. `application.js`,
+2. `vendor.js`,
+3. `963eb32907744d9a0d6b98127162808f.svg` (fingerprint will be different),
```

Comment:
Sure, but "fingerprint" means nothing if you don't know what this is about. How about *both* - "The file name (also known as *fingerprint*) will be different)